### ISSUE TITLE
Crash multi-line strings

### ISSUE THREAD

**Issue by chillman79** [JujuAdams/ScribbleJunior#13]
Hello!

I've encountered this bug a few times using ScribbleJr. 

**GameMaker Version**: v2024.6 (Runtime 2024.6.1)  but also tested in v2024.8 (Runtime 2024.8.1)
**Platform:** Windows
**ScribbleJr**: 1.5.0

I have managed to reproduce the problem with the example project. The error appears when you use multi-line strings. 

In the sample project, oTest object -> modify the line
```gml
testString = 'Sphinx of black quartz,\njudge my vow!';
``` 
By: 
```gml
testString = @"Sphinx 
of black quartz,\n
judge my vow!
";
```

The error I get is:

```
___________________________________________
############################################################################################
ERROR in action number 1
of Draw Event for object oTest:
Push :: Execution Error - Variable Index [0] out of range [0] - -6.__stringArray(100063,0)
 at gml_Script___TickLine@anon@2503@__ScribblejrClassBaker@__ScribblejrClassBaker (line 92) -             var _char = __stringArray[__glyph];
############################################################################################
gml_Script___TickLine@anon@2503@__ScribblejrClassBaker@__ScribblejrClassBaker (line 92)
gml_Script___BakeVertexBuffer@anon@676@__ScribblejrClassBase@__ScribblejrClassBase (line 29) -         if (__vertexBaker.__tickMethod())
gml_Script___BakeVertexBufferTimed@anon@330@__ScribblejrClassBase@__ScribblejrClassBase (line 20) -         __BakeVertexBuffer();
gml_Script___Draw@anon@2821@__ScribblejrClass@__ScribblejrClass (line 141) -         if (SCRIBBLEJR_AUTO_BAKE) __BakeVertexBufferTimed();
gml_Script_Draw@anon@131@__ScribblejrClassWrapper@__ScribblejrClassWrapper (line 9) -         __element.Draw(_x, _y, _colour, _alpha, _sdfEffects);
gml_Object_oTest_Draw_0 (line 55) -             _element.Draw(x, y, un, un, _sdfEffects);

```

Thank you!

**Comment by JujuAdams** [comment]
What version of Scribble Junior are you using? What version of GameMaker are you using? What platform are you testing on? Are you able to reproduce this using the repo test project and, if so, what are the steps to reproduce the crash?

Please post the full error message without any truncation.

**Comment by chillman79** [comment]
My bad, I've updated the issue description. It seems to be something with multiline strings.

**Comment by JujuAdams** [comment]
Alrighty, I'll take a look when I get a moment.

**Comment by JujuAdams** [comment]
Fixed in 1.6.2 beta https://github.com/JujuAdams/ScribbleJunior/releases/tag/1.6.2.beta
